lib/internal/Magento/Framework/Module/PackageInfo.php

```diff
@@ -85,9 +95,9 @@ private function load()
                 $key = $moduleDir . '/composer.json';
                 if (isset($jsonData[$key]) && $jsonData[$key]) {
                     try {
-                        $packageData = \Zend_Json::decode($jsonData[$key]);
-                    } catch (\Zend_Json_Exception $e) {
-                        throw new \Zend_Json_Exception(
+                        $packageData = $this->serializer->unserialize($jsonData[$key]);
+                    } catch (\InvalidArgumentException $e) {
```

Comment:
Do you mean specific exception for Json or Serialization?
I assume this specific fix is fine, since there is no major change.